react/components/EXPERIMENTAL_Table/DataTable/Cell.tsx

```diff
@@ -35,19 +35,31 @@ const Cell: FC<CellProps> & CellComposites = ({
   className: classNameProp = '',
   active = false,
   link = false,
+  sticky = false,
+  header,
 }) => {
   const { hover, ...events } = useHover()
   const className = classNames(
-    'v-mid ph3 pv0 tl bb b--muted-4',
+    'v-mid ph3 pv0 tl bb b--muted-4 bg-base',
     classNameProp,
     {
       pointer: onClick,
       'hover-c-link hover-bg-muted-5': link,
       'c-on-base': active,
+      'top-0 z3': sticky && header,
+      z1: !sticky,
     }
   )
+
   return (
-    <Tag {...events} onClick={onClick} style={{ width }} className={className}>
+    <Tag
+      {...events}
+      onClick={onClick}
+      style={{
+        position: sticky ? 'sticky' : 'static',
```

Comment:
I don't know if we should be doing this using css because of the [support of browsers](https://caniuse.com/#search=sticky). In the store we had to do something similar, so we create this behavior with js. https://github.com/vtex-apps/sticky-layout cc @kaisermann

Reply:
Yeah, support for actual tables is not great, but it seems you're not using `<tr/td/thead>`, right?

Reply:
I only use the `sticky` position on the `<th/>`s.  That's the only table element that works well with this feature on stable browsers. Tested on chrome, safari, firefox, and edge (for mac).